lib/yabeda/porta.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+# Based on https://github.com/yabeda-rb/yabeda-rails
+
+require "yabeda"
+
+module Yabeda
```

Comment:
I would opt not to put code under a namespace we don't own unless necessary. For example `Metrics::Yabeda` would be my suggestion.